common/models/role.js

```diff
@@ -210,26 +210,42 @@ module.exports = function(Role) {
         if (callback) callback(null, matches(ownerId, userId));
         return;
       } else {
```

Comment:
That way you allow multiple owner only when it's not base on the `userId` / `owner` field ... see https://github.com/strongloop/loopback/pull/3106/files for my implementation